pandas/core/groupby/ops.py

```diff
@@ -597,7 +606,7 @@ def _cython_operation(self, kind, values, how, axis, min_count=-1, **kwargs):
 
         if how in self._name_functions:
             # TODO
-            names = self._name_functions[how]()
+            names = self._name_functions[how]()  # type: Optional[List[str]]
```

Comment:
the comment above this line is really annoying

Reply:
Yea I suppose not very helpful...I'll remove